scripts/htmlDependencies.R

```diff
@@ -19,7 +19,7 @@ withr::local_temp_libpaths()
 ignore <- capture.output({
   pak::pkg_install(c(
     "rstudio/bslib@main",
-    "rstudio/shiny@main",
```

Comment:
TODO: get https://github.com/rstudio/shiny/pull/4040 merged and switch this back to `main`